src/server/routes/index.js

```diff
@@ -45,11 +45,11 @@ router.get('/', async (req, res, next) => {
 	const {orm} = req.app.locals;
 	const numRevisionsOnHomepage = 9;
 
-	function render(entities) {
+	function render(results) {
 		const props = generateProps(req, res, {
 			homepage: true,
-			recent: _.take(entities, numRevisionsOnHomepage),
-			requireJS: Boolean(res.locals.user)
+			requireJS: Boolean(res.locals.user),
+			results: _.take(results, numRevisionsOnHomepage)
 		});
```

Comment:
I still think that call to generateProps function is not useful. I think you only end up needing props.results, which is just as good as the original results array.
In any case, the rest of the props are incorrect: homepage shouldn't be true and I don't think you need requireJS.

Reply:
I didn't change this part, it was like this before also. I just renamed it from recent to result

Reply:
I think you can just remove the `homepage` and `requireJS` properties.

Reply:
If homepage is not true, then it shows search bar in the page. And for index page we don't want that because there is already a search bar in it